spec.html

```diff
@@ -8444,7 +8444,7 @@ <h1>Module Namespace Exotic Objects</h1>
               [[Exports]]
             </td>
             <td>
-              List of String
+              List of String | *null*
```

Comment:
Doesn't the need for all this | null/extensibility stuff go away?

Reply:
Ah, never mind, no, this is how CJS modules are directly (it is just never that a ESM is like that), right?

Reply:
Yes, the pending state still stands - this is for the case on the 7th slide of https://docs.google.com/presentation/d/1--k5tJmhXJeLlUOmBdHGK3my9VCbZVL0MF8OrTeqS0I/edit?usp=sharing.